docs/docs/guides/FRAME_PROCESSORS_CREATE_OVERVIEW.mdx

```diff
@@ -123,7 +126,7 @@ Which will throw a JS-error:
 const frameProcessor = useFrameProcessor((frame) => {
   'worklet'
   try {
-    const codes = scanCodes(frame, true)
+    const codes = scanCodes(frame, { someString: true })
```

Comment:
Maybe let's use more accurate examples? E.g. `codeType: 'qr' | 'barcode'`.

Or maybe we wanna refactor this to object detection with `objectType: 'banana'`. I'll think about this

Reply:
Actually we are using scanFaces in other examples, no?

Reply:
In the overview page there're multiple examples, but that line is actually about passing some argument that will raise an exception. However I'm open to changing the name and value to be more accurate :D

Reply:
So I changed it to `codeType: string` on Java side and for raising exception there is `codeType: 1234` on JS side (so String != Int)